src/index.html

```diff
@@ -4,10 +4,315 @@
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#about">About Us</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#gallery">Gallery</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#consultation">Consultation</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#testimonials">Testimonials</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#contact">Contacts</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="tel:+1 234 555-55-55">Call to order</a>
+            </li>
+          </ul>
+        </nav>
+
+        <img
+          class="top-line__logo"
+          src="images/logo.svg"
+          alt="miami logo"
+        >
+
+        <a class="top-line__tel" href="tel:+1 234 555-55-55"></a>
+      </div>
+
+      <h1 class="header__title">Miami<br>Condo&nbsp;Kings</h1>
+    </header>
+
+    <main class="main">
+      <section id="about" class="about">
+        <h2 class="about__title">
+          Miami best<br>
+          Real estate <a href="https://goo.gl/maps/FryykxwpkdQ8YkXHA">here</a>
+        </h2>
+
+        <article class="about__info">
+          <h3>Over 50 years experience in love.</h3>
+
+          <p>
+            Mazel and VA Tsukkerman in his "Analysis of musical works."
+            Pointillism, which originated in the music of the early twentieth
+            century, microforms, found a distant historical.
+          </p>
+
+          <p>
+            The market leader in real estate since 2003. Awards Real Estate
+            Company, 2010, 2011, 2013. More than 200 satisfied customers
+            premium segment. We focus only on new buildings and high-level
+            doskanalno know this area better than other specialists.
+            Working directly with developers - so have more useful information
+            about the profitable deals.
+          </p>
+
+          <p>
+            We will never be late!<br>
+            Guaranteed secure transaction - 100%
+          </p>
+        </article>
+      </section>
+
+      <section id="gallery" class="gallery">
+        <h2 class="gallery__title">Gallery</h2>
+
+        <div class="gallery__block gallery__block--1">
+          <img
+            class="gallery__photo"
+            src="images/photos/1.jpg"
+            alt="Marina Palms"
+          >
+
+          <img
+            class="gallery__photo"
+            src="images/photos/2.jpg"
+            alt="Miami Beach"
+          >
+        </div>
+
+
+        <p class="gallery__inscription">
+          <a href="https://goo.gl/maps/HPqWdhYPH9ZzVgeJ6">Marina Palms</a>
+          / North Miami Beach, FL 33162
+        </p>
+
+        <div class="gallery__block gallery__block--2">
```

Comment:
```suggestion
        <div class="gallery__block">
```
redundant. You don't use any styles for `block--2`